streams/src/test/java/org/apache/kafka/streams/state/internals/CompositeReadOnlyWindowStoreTest.java

```diff
@@ -172,8 +172,8 @@ public void shouldFetchKeyRangeAcrossStores() {
                 ReadOnlyWindowStoreStub<>(WINDOW_SIZE);
         stubProviderTwo.addStore(storeName, secondUnderlying);
         underlyingWindowStore.put("a", "a", 0L);
-        secondUnderlying.put("b", "b", 0L);
-        List<KeyValue<Windowed<String>, String>> results = StreamsTestUtils.toList(windowStore.fetch("a", "b", 0, 1));
+        secondUnderlying.put("b", "b", 10L);
+        List<KeyValue<Windowed<String>, String>> results = StreamsTestUtils.toList(windowStore.fetch("a", "b", 0, 10));
         assertThat(results.size(), equalTo(2));
```

Comment:
Shouldn't we be asserting the entries in `results`?

Reply:
Yes that would be better